chef/cookbooks/neutron/recipes/l3.rb

```diff
@@ -156,6 +156,16 @@
   )
 end
 
+template "/etc/neutron/dnsmasq-neutron.conf" do
+  source "dnsmasq-neutron.conf.erb"
+  owner "root"
+  group "root"
+  mode "0644"
+  variables(
+    :low_mtu => node[:neutron][:ml2_type_drivers].include?('gre')
```

Comment:
It's an attribute from the proposal (marked as "required" in the schema). So yes, assuming that it's allways there should be fine. (It might be empty though, but doesn't matter here).